src/main/java/frc/lib/util/swerveUtil/COTSNeoSwerveConstants.java

```diff
@@ -0,0 +1,204 @@
+package frc.lib.util.swerveUtil;
+
+import edu.wpi.first.math.util.Units;
+
+/* Contains values and required settings for common COTS swerve modules. */
+public class COTSNeoSwerveConstants {
+    public final double wheelDiameter;
+    public final double wheelCircumference;
+    public final double angleGearRatio;
+    public final double driveGearRatio;
+    public final double angleKP;
+    public final double angleKI;
+    public final double angleKD;
+    public final double angleKF;
+    public final boolean driveMotorInvert;
+    public final boolean angleMotorInvert;
+    public final boolean canCoderInvert;
+
+    public COTSNeoSwerveConstants(double wheelDiameter, double angleGearRatio, double driveGearRatio, double angleKP, double angleKI, double angleKD, double angleKF, boolean driveMotorInvert, boolean angleMotorInvert, boolean canCoderInvert){
```

Comment:
What units is the wheel diameter given in?

Reply:
This whole file is not used in the codebase - COTSFalconSwerveConstants is. Therefore, I'm going to delete it.